railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt

```diff
@@ -15,9 +15,11 @@ Rails.application.configure do
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
   if Rails.root.join('tmp', 'caching-dev.txt').exist?
+    <%- unless options.api? -%>
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
 
+    <%- end -%>
     config.cache_store = :memory_store
     config.public_file_server.headers = {
       'Cache-Control' => "public, max-age=#{2.days.to_i}"
```

Comment:
Is the setting of `config.action_controller.perform_caching = false` left at following is intentional?

Reply:
yes
Because `config.action_controller.perform_caching` is `true` by default. So for the cases we do want to disable it, we still need to set it to `false` explicitly.

Reply:
I understood that the config was removed because of the module which is not used by default.
If it is not necessary because it is the same as the default value, why is it removed only in the case of API only?

Reply:
For documentation purpose.

Reply:
I see. Thanks.